Core/StringExtension.swift

```diff
@@ -50,6 +50,13 @@ extension String {
         return self
     }
 
+    public func dropSuffix(suffix: String) -> String {
+        if hasSuffix(suffix) {
+            return String(dropLast(suffix.count))
+        }
+        return self
+    }
```

Comment:
Yes, I used it in previous commits but not in latest version. I thought it could be useful in the future so left it there. Do you want to clean it? :)

Reply:
If the code is "dead", then I'd clean it. :)